include/blosc2.h

```diff
@@ -1819,6 +1825,19 @@ BLOSC_EXPORT blosc2_schunk* blosc2_schunk_open(const char* urlpath);
  */
 BLOSC_EXPORT blosc2_schunk* blosc2_schunk_open_offset(const char* urlpath, int64_t offset);
 
+/**
+ * @brief Open an existing super-chunk that is on-disk (frame) in memory-mapped mode. No in-memory copy is made.
+ *
+ * @param urlpath The file name.
+ *
+ * @param offset The frame offset.
+ * 
+ * @param mmap_mode The mode for the memmory mapping similar to Numpy's np.memmap (https://numpy.org/doc/stable/reference/generated/numpy.memmap.html). Set to r if the super-chunk should only be read. See #blosc2_stdio_mmap struct for details.
+ *
+ * @return The new super-chunk.  NULL if not found or not in frame format.
+ */
+BLOSC_EXPORT blosc2_schunk* blosc2_schunk_open_offset_mmap(const char* urlpath, int64_t offset, const char* mmap_mode);
+
```

Comment:
This is the only mmap specific API introduced.  May I suggest to add `blosc2_schunk_open_offset_udio` instead?  This would work more generally for other udio.  OTOH, I agree that `blosc2_schunk_open_offset_mmap` is handy, but having it and not `blosc2_schunk_open_mmap` is not consistent (i.e. the latter should be added too).  A bit undecided here...  Thoughts?

Reply:
I agree. The `blosc2_schunk_open_udio` function can already be used for mmap and it only makes sense to also have a `blosc2_schunk_open_offset_udio` counterpart (especially since this is needed for Python). I removed the `blosc2_schunk_open_offset_mmap` function.